client/internal/bootstrap/grpc_test.go

```diff
@@ -7,143 +7,163 @@ import (
 	"crypto/x509"
 	"os"
 	"path/filepath"
+	"testing"
 	"time"
 
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
 	"google.golang.org/grpc/test/bufconn"
 
 	"github.com/Azure/aks-secure-tls-bootstrap/client/internal/testutil"
+	"github.com/stretchr/testify/assert"
 )
 
-var _ = Describe("grpc", Ordered, func() {
+func TestGRPC(t *testing.T) {
 	var (
 		clusterCACertPEM []byte
 	)
-
-	BeforeAll(func() {
-		var err error
-		clusterCACertPEM, _, err = testutil.GenerateCertPEM(testutil.CertTemplate{
-			CommonName:   "hcp",
-			Organization: "aks",
-			IsCA:         true,
-			Expiration:   time.Now().Add(time.Hour),
-		})
-		Expect(err).To(BeNil())
+	var err error
+	clusterCACertPEM, _, err = testutil.GenerateCertPEM(testutil.CertTemplate{
```

Comment:
```suggestion
	clusterCACertPEM, _, err := testutil.GenerateCertPEM(testutil.CertTemplate{
```

and remove the `var ( ... )`, and `var err error`